compiler/crates/relay-typegen/tests/generate_typescript/fixtures/jsdoc-query.expected

```diff
@@ -0,0 +1,55 @@
+==================================== INPUT ====================================
+query FooQuery {
+  me {
+    fullName
+    dynamicFriends(first: 10) {
+      edges {
+        node {
+          firstName
+        }
+      }
+    }
+  }
+}
+==================================== OUTPUT ===================================
+export type FooQuery$variables = Record<PropertyKey, never>;
+export type FooQuery$data = {
+  readonly me: {
+    /**
+     * List the user's friends based on previous interactions.
+
```

Comment:
Can we handle multi line comments better? By convention, each line should indented to the right level and start with a space + asterisk + space.

Reply:
hum correct 🤔 ...
i'll check what i can do with my clunky  🦀 skills

Reply:
ahah it turns out to be not as hard as i tought 😅 ... i think it's better now 🚀 ...